### DISCUSSION TITLE
GEPHI ON MACBOOK

### DISCUSSION THREAD

**Discussion by SHRINIVASDHARMADHIKARI** [gephi/gephi#2843] [Q&A]
Dear Friends

I am trying to use a Macbook with Gephi.
Firstly, nothing appears in the Overview and Data Laboratory.
Even after loading data appropriately, there is no trace of action in the Overview and Data Laboratory. 
Also, I get an instruction saying Processing "miserables.gexf" which is active continuously

Please help

Regards

SHRINIVAS

**Comment by andrewcrook** [comment]
try installing it via homebrew

`brew install  gephi`

working on my M1 16GB under macOS 14
